content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -475,16 +475,15 @@ Kubelets prior to v1.24 don't enforce the pod OS field, and if a cluster has nod
 {{< /note >}}
 
 ### Restricted Pod Security Standard changes
-Another important change, made in Kubernetes v1.25 is that the  _restricted_ Pod security
+Another important change, made in Kubernetes v1.25 is that the  _Restricted_ policy
 has been updated to use the `pod.spec.os.name` field. Based on the OS name, certain policies that are specific
 to a particular OS can be relaxed for the other OS.
 
-
 #### OS-specific policy controls
 Restrictions on the following controls are only required if `.spec.os.name` is not `windows`:
 - Privilege Escalation
 - Seccomp
-- Linux Capabilities
+- Capabilities
```

Comment:
Why this change?

Reply:
I changed to align with the name of the control (`Capabilities`) in the Baseline table.

Reply:
Let's emphasize what kind of capability this is. The table has to be a little more terse to help layout.

Reply:
Sure